src/lib/MapLibre.svelte

```diff
@@ -144,6 +159,10 @@
       zoom = ev.target.getZoom();
       bounds = ev.target.getBounds();
       dispatch('moveend', { ...ev, map: $mapInstance });
+      if (hash) {
+        let location = window.location.href.replace(/(#.+)?$/, getViewportHash($mapInstance));
+        window.history.replaceState(window.history.state, '', location);
```

Comment:
Do we want to carry forward https://github.com/maplibre/maplibre-gl-js/blob/5d7e6d52000a8569ac2308a9aef14c98933eb0d8/src/ui/hash.ts#L126?

Reply:
Probably fine to skip, IE11 is long gone :)